cds_ils/importer/importer.py

```diff
@@ -90,12 +89,9 @@ def _validate_provider(self):
         agency_code = self.json_data.get("agency_code")
         if not agency_code:
             raise UnknownProvider
-        assert (
-            self.json_data["agency_code"]
-            == current_app.config["CDS_ILS_IMPORTER_PROVIDERS"][
-                self.metadata_provider
-            ]["agency_code"]
-        )
+        if not agency_code == current_app.config["CDS_ILS_IMPORTER_PROVIDERS"][
+                self.metadata_provider]["agency_code"]:
+            raise InvalidProvider
```

Comment:
small readability improvement:
```suggestion
        agency_code = self.json_data.get("agency_code")
        config_agency_code = current_app.config["CDS_ILS_IMPORTER_PROVIDERS"][
                self.metadata_provider]["agency_code"]
        if not agency_code:
            raise UnknownProvider
        if agency_code != config_agency_code:
            raise InvalidProvider
```